src/error.js

```diff
@@ -54,18 +54,35 @@ let globalExponentialBackoff = function(work) {
  * elements instead of stringification.
  * @param {*} error
  * @param {!Element=} opt_associatedElement
+ * @return {!Error}
  */
 export function reportError(error, opt_associatedElement) {
-  if (!self.console) {
-    return;
+  // Convert error to the expected type.
+  let isValidError;
+  if (error) {
+    if (typeof error == 'string') {
```

Comment:
Maybe check for `object` and `!null` instead? Errors could also be numbers and other crap.